packages/api-client/src/components/EnvironmentSelector/EnvironmentSelector.vue

```diff
@@ -0,0 +1,102 @@
+<script setup lang="ts">
+import { useWorkspace } from '@/store/workspace'
+import {
+  ScalarButton,
+  ScalarDropdown,
+  ScalarDropdownItem,
+  ScalarIcon,
+} from '@scalar/components'
+import { computed } from 'vue'
+import { useRouter } from 'vue-router'
+
+const {
+  environments,
+  activeWorkspace,
+  workspaceMutators,
+  activeEnvironment,
+  isReadOnly,
+} = useWorkspace()
+
+const router = useRouter()
+
+const updateSelected = (uid: string) => {
+  workspaceMutators.edit(activeWorkspace.value.uid, 'activeEnvironmentId', uid)
+}
+
+const createNewEnvironment = () =>
+  router.push(`/workspace/${activeWorkspace.value.uid}/environment`)
+
+const envs = computed(() => {
+  return Object.values(environments).filter((env) => env.uid !== 'default')
+})
```

Comment:
```suggestion
const envs = computed(() => Object.values(environments).filter((env) => env.uid !== 'default'))
```
don't need returns with arrow functions ;)